projects/aws/lib/aws-stack.ts

```diff
@@ -227,5 +231,60 @@ export class EditionsStack extends cdk.Stack {
                 sourceArn: publishedBucket.bucketArn,
             },
         )
+
+        //Archiver step function
+
+        const archiverStateMachine = stepFunction(this, {
```

Comment:
Again, in the context, it's not clear what step function is doing, I think it'd be clearer if it had the word `archive` somewher in it ...